README.md

```
@@ -23,9 +22,29 @@ A comprehensive Extract, Transform, Load (ETL) pipeline for business analysis an
 ## 📁 Repository Structure
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)